### ISSUE TITLE
Adding seed for VB

### ISSUE THREAD

**Issue by Skoricius** [pyGSTio/pyGSTi#253]
`PeriodicMirrorCircuitDesign` in `vb.py` currently does not support seeding for reproducible results. But `create_random_germpower_mirror_circuits` method that it uses supports it. 

To fix this, it would just be down to adding the seed to the `__init__` method and passing it to `create_random_germpower_mirror_circuits` (line 259 in `vb.py`):

```python
            circtemp, outtemp, junk = _rc.create_random_germpower_mirror_circuits(
                pspec, abs_clifford_compilations, depths, qubit_labels=qubit_labels, localclifford=localclifford,
                paulirandomize=paulirandomize, interacting_qs_density=samplerargs[0],
                fixed_versus_depth=fixed_versus_depth,
                seed=seed)
```

**Comment by sserita** [comment]
Thanks for reporting this. This was also noted as a subissue in #252. Hopefully should get to this soon.

**Comment by sserita** [comment]
Closed as this is now in 0.9.11 release
